backend/src/main/java/com/zeros/backend/repositories/RestaurantAffiliateRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.zeros.backend.repositories;
+
+
+import com.zeros.backend.models.RestaurantAffiliate;
+
+public interface RestaurantAffiliateRepository extends Repository<RestaurantAffiliate>{
+    RestaurantAffiliate findByRestaurantId(Long restaurantId);
+}
```

Comment:
hi @JumperSpirit  , we have already a repository to manage Restaurant entity
RestaurantRepository ; you can use it

Reply:
I prefere separate subscribtion and accepting donation what do you think ??